### PR TITLE
Аdds oz beacon proxy tests

### LINKED ISSUE #599: Add OZ beacon proxy example

### PR DESCRIPTION
**Pull request by konstantinabl**
**Description**:

This PR adds test for the beacon proxy OZ contracts 
**Related issue(s)**:

Fixes #599

### CODE REVIEW

File: test/beacon-proxy/beaconProxy.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
+/*-
```
Comment (quiet-node) [inline test/beacon-proxy/beaconProxy.js:1]: Please move this folder into `test/solitiy/oz/` folder

File: test/solidity/oz/transparent-upgradeable-proxy/transparentProxy.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+/*-
```
Comment (quiet-node) [inline test/solidity/oz/transparent-upgradeable-proxy/transparentProxy.js:1]: duplicated

**Review by quiet-node — CHANGES_REQUESTED** [review]
LG but some comments

File: contracts/solidity/oz/TransparentUpgradeableProxy/Box.sol [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+ 
+contract Box {
```
[the remark this answers is not in the document]
Comment (konstantinabl) [reply contracts/solidity/oz/TransparentUpgradeableProxy/Box.sol]: I rebase onto upgradeable proxy branch cause i needed some changes. Fixed now

**Review by quiet-node — APPROVED** [review]
LGTM

**Review by georgi-l95 — APPROVED** [review]
👁️ 👍 👉 👨‍💻

**Pull request merged**